test/WebJobs.Script.Tests.Integration/EndToEndTestFixture.cs

```diff
@@ -64,7 +70,12 @@ protected EndToEndTestFixture(string rootPath, string testId)
             var fastLogger = new FunctionInstanceLogger(funcLookup, new MetricsLogger());
             config.HostConfig.AddService<IAsyncCollector<FunctionInstanceLogEntry>>(fastLogger);
 
-            Host = ScriptHost.Create(ScriptHostEnvironmentMock.Object, EventManager, config, _settingsManager);
+            IProxyClient proxyClient = null;
```

Comment:
Rather than make all these modifications to the base test file used by all the tests, can't you just put all this proxy specific stuff in your new proxies test class? None of the other tests need it.